src/misc/inc/1DQuadrature.h

```diff
@@ -44,8 +45,8 @@ namespace QUESO {
 
     Base class for numerical integration via quadrature rules of one-dimensional functions.
 */
-class
-Base1DQuadrature {
+class Base1DQuadrature : public QuadratureBase
```

Comment:
Is `Base1DQuadrature` intended to be abstract?  There are no pure virtual functions in here.